docs/exult_intrinsics.txt

```diff
@@ -3237,6 +3237,12 @@ obj->obj_sprite_effect(int sprite, int rel_x, int rel_y, int vel_x, int vel_y,
               the sprite frame will go through all of its frames. If negative,
               the sprite will go through all of its frames *once*.
 
+string UI_play_scene(string text)
```

Comment:
Also, the intrinsic as defined in [usecode/intrinsics.cc](https://github.com/exult/exult/pull/756/files#diff-36c9a429bcde9092290a67190351652dcb6ba679aac1ab53918bbd969411795a) does not return anything, and so should be `void` return type.